README.md

```diff
@@ -47,6 +47,7 @@ We rely on different `django` and `mypy` versions:
 
 | django-stubs | mypy version | django version | python version
 | ------------ | ---- | ---- | ---- |
+| 1.5.0 | ^0.770 | 3.0.x | ^3.7.7
```

Comment:
1. `1.4.0` is missing
2. python version is always `^3.6`
3. django version is `2.2 || 3.x`
4. `mypy` should be `0.780`

Reply:
Please check if now it's correct, I was basing on https://github.com/typeddjango/django-stubs/releases